main/msmtp/patches/tests.patch

```diff
@@ -0,0 +1,140 @@
+these files are missing from the 1.8.27 dist tarball
```

Comment:
is there a non-dist tarball then? i don't think patching back in every test file is a good idea

Reply:
looks ok then